MyProject/src/main/java/com/netcracker/controller/InventoryController.java

```diff
@@ -0,0 +1,265 @@
+package com.netcracker.controller;
+
+import com.netcracker.exceptions.ObjectNotFoundException;
+import com.netcracker.model.documents.HardwareComponent;
+import com.netcracker.model.documents.Interface;
+import com.netcracker.model.documents.NetworkElement;
+import com.netcracker.model.documents.SubInterface;
+import com.netcracker.model.edges.IntToSub;
+import com.netcracker.model.edges.Link;
+import com.netcracker.model.edges.NeToHw;
+import com.netcracker.model.edges.NeToInterface;
+
+import com.netcracker.repository.documents.HwRepository;
+import com.netcracker.repository.documents.InterfaceRepository;
+import com.netcracker.repository.documents.NeRepository;
+import com.netcracker.repository.documents.SubRepository;
+import com.netcracker.repository.edges.IntToSubRepository;
+import com.netcracker.repository.edges.LinkRepository;
+import com.netcracker.repository.edges.NeToHwRepository;
+import com.netcracker.repository.edges.NeToInterRepository;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.NoSuchElementException;
+
+@RestController
+@RequestMapping("/data")
+@Api(value = "Network system")
+@RequiredArgsConstructor
+public class InventoryController {
+
+    private final InterfaceRepository interfaceRepository;
+    private final HwRepository hwRepository;
+    private final NeRepository neRepository;
+    private final NeToHwRepository neToHwRepository;
+    private final NeToInterRepository neToInterRepository;
+    private final LinkRepository linkRepository;
+    private final SubRepository subRepository;
+    private final IntToSubRepository intToSubRepository;
```

Comment:
I would recomment to encapsulate all of the repositories inside separate crud inventory service and implement all get/post logic in there. Please check https://github.com/dmfe/network-inventory/blob/master/src/main/java/com/example/inventoryapp/services/impl/InventoryCRUDService.java as example